webpack/assets/javascripts/react_app/components/notifications/index.js

```diff
@@ -8,9 +8,11 @@ import { groupBy, some, isUndefined } from 'lodash';
 
 class notificationContainer extends React.Component {
   componentDidMount() {
-    const { getNotifications, data: { url } } = this.props;
+    const { startNotificationsPolling, data: { url }, isPolling } = this.props;
 
-    getNotifications(url);
+    if (!isPolling) {
+      startNotificationsPolling(url);
```

Comment:
shouldn't this be defined at the pulling action? why would the component know about it (or care)?